validation/score.py

```diff
@@ -26,10 +31,12 @@ def parse_args():
         help="Path to the validation CSV file",
     )
     parser.add_argument(
-        "--latitude-resolution", default=0.005, type=float, help="Latitude resolution"
+        "--n-samples", default=None, type=int, help="Number of samples to use"
     )
     parser.add_argument(
-        "--longitude-resolution", default=0.005, type=float, help="Longitude resolution"
```

Comment:
question: why are we getting rid of this arg?

Reply:
see above